File: LOGICA 2/ATIVIDADE4/Exercicio2.java
import java.util.Scanner;
public class Exercicio2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Insira um número inteiro");
        int num = input.nextInt();
        if((num % 2) == 0){
            System.out.println(num+" é par");
        }else{
            System.out.println(num+" é impar");
        }
    }
}

File: python/Exercicio2.py


nums = []
for i in range(5):
    nums.append(input("Insira um número inteiro"))
    
nums = [int(x) for x in nums]

for num in nums:
    if num % 2 == 0:
        print(f'{num} é par')
    else:
        print(num)

# Crie um vetor que leia 
# valores inteiros , e mostre os 
# valores armazenados , e 
# mostre quais são pares